ndarray/src/main/java/org/tensorflow/ndarray/NdArrays.java

```diff
@@ -90,13 +96,32 @@ public static ByteNdArray ofBytes(Shape shape) {
    * @param shape shape of the array
    * @param buffer buffer to wrap
    * @return new byte N-dimensional array
-   * @throws IllegalArgumentException if shape is null, has unknown dimensions or has size bigger
-   *                                  in the buffer size
+   * @throws IllegalArgumentException if shape is null, has unknown dimensions or has size bigger in
+   *     the buffer size
    */
   public static ByteNdArray wrap(Shape shape, ByteDataBuffer buffer) {
     return ByteDenseNdArray.create(buffer, shape);
   }
 
+  /**
+   * Creates a Sparse array of byte values
+   *
+   * @param indices A 2-D LongNdArray of shape {@code [N, ndims]}, that specifies the indices of the
+   *     elements in the sparse array that contain nonzero values (elements are zero-indexed). For
+   *     example, {@code indices=[[1,3], [2,4]]} specifies that the elements with indexes of {@code
+   *     [1,3]} and {@code [2,4]} have nonzero values.
+   * @param values A 1-D ByteNdArray of shape {@code [N]}, which supplies the values for each
+   *     element in indices. For example, given {@code indices=[[1,3], [2,4]]}, the parameter {@code
+   *     values=[18, 3]} specifies that element {@code [1,3]} of the sparse NdArray has a value of
+   *     {@code 18}, and element {@code [2,4]} of the NdArray has a value of {@code 3}.
+   * @param shape the shape of the dense array represented by this sparse array.
+   * @return the byte sparse array.
+   */
+  public static ByteSparseNdArray sparseTensorOf(
```

Comment:
In the Java NdArray space, we don't talk about "tensors" but simply about "arrays". So I would suggest to rename this method and the following by simply `sparseOf`

Reply:
Ok, renamed all to `sparseOf`